e2e/BKTClient.spec.ts

```diff
@@ -50,11 +50,12 @@ suite('e2e/BKTClientTest', () => {
     const detail = client.evaluationDetails(FEATURE_ID_STRING)
 
     expect(detail).toBeEvaluation({
-      id: 'feature-js-e2e-string:4:bucketeer-js-user-id-1',
+      id: 'feature-js-e2e-string:5:bucketeer-js-user-id-1',
       featureId: FEATURE_ID_STRING,
-      featureVersion: 4,
+      featureVersion: 5,
```

Comment:
Because I added the variation name on the console UI, it increased the feature version.
Same for the other e2e tests.